docs/concepts/abstractions/pod.md

```diff
@@ -0,0 +1,74 @@
+---
+---
+
+{% capture overview %}
+This page provides an overview of `Pod`, the smallest deployable object in the Kubernetes object model.
+{% endcapture %}
+
+* TOC
+{:toc}
+
+{% capture body %}
+### Understanding Pods
+
+A *Pod* is the basic building block of Kubernetes--the smallest and simplest unit in the Kubernetes object model that you create or deploy. A Pod represents a running process on your cluster.
+
+A Pod encapsulates an application container (or, in some cases, multiple containers), storage resources, and options that govern how the container(s) should run. A Pod represents a unit of deployment: *a single workload in Kubernetes*, which might consist of either a single application or a small number of applications that are tightly coupled and that share resources.
+
+> [Docker](https://www.docker.com) is the most common container runtime used in a Kubernetes Pod, but Pods support other container runtimes as well.
+
+Pods are employed a number of ways in a Kubernetes cluster, including:
+
+* **Pods that run a single application container**. The "one-application per Pod" model is the most common Kubernetes use case; in this case, you can think of a Pod as a wrapper around a single application, and Kubernetes manages the Pods rather than the containers directly.
+* **Pods that run multiple application containers that need to work together**. Pods can support multiple application containers that are tightly coupled and need to share resources. You can think of these applications as forming a *single cohesive unit of service*. The Pod wraps them together with shared resources as a single managable entity.
+
+Pods typically *do not* model multiple instances of the same application container. Instead, you can have Kubernetes maintain separate Pods for each instance you want to run, usually managed by a Controller. See [Pods and Controllers](#pods-and-controllers) for more information.
+
+#### How Pods Manage Containers
+
+Pods are designed to support multiple cooperating processes (as application containers) that form a cohesive unit of service. The containers in a Pod are automatically co-located and co-scheduled on the same phyiscal or virtual machine in the cluster. The containers can share resources and dependencies, communicate with one another, and coordinate when and how they are terminated.
```

Comment:
I think we need to be more concrete when we talk about this use case and keep it simple.  

Maybe include something like

> This can be used to run your application as a container in a Pod, and then run a second "side-car" container to sync files remotely for your application to consume (e.g. from GitHub or some other source).